perf/inferenceql/query/perf.clj

```diff
@@ -0,0 +1,142 @@
+(ns inferenceql.query.perf
+  (:require [clj-async-profiler.core :as prof]
+            [clojure.java.browse :as browse]
+            [clojure.string :as str]
+            [criterium.core :as crit]
+            [inferenceql.query.strict :as strict]
+            [medley.core :as medley]))
+
+(def ^:dynamic *default-limit* 10000)
+
+(defn- str-listify
```

Comment:
looks like this is here because you want to allow users to specify a bare string for `col-var-list`. Might it be better to insist on a list, since that is part of the parameter name?

Reply:
I'd like to support one-off, single queries as well, especially for interactive use at the REPL.

Unfortunately, I've never heard of a great name to express that something could be one or many. `str-or-list-of-strs` is accurate but awkward. If parens were allowed, we could call it `str(s)`...

I can at least remove the `-list` part, and just call it `col-vars`, unless you have a better name.